videos/views.py

```diff
@@ -36,3 +38,20 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
             video_job = VideoJob.objects.get(job_id=detail.get("jobId"))
             update_video_job(video_job, detail)
         return Response(status=200, data={})
+
+
+class TranscriptJobView(GenericAPIView):
+    """ Webhook endpoint for transcript completion notifications from 3play"""
+
+    permission_classes = (AllowAny,)
```

Comment:
Not sure if it's worth it, but assuming 3play allows, could add a bearer token authorization to the 3play request and then use the `BearerTokenPermission` class to make sure it's the same value as `settings.API_BEARER_TOKEN` to filter out any bogus requests.

Reply:
Threeplay doesn't support bearer tokens, just a url. I added a url key as an api parameter